sonar-html-plugin/src/main/java/org/sonar/plugins/html/api/HtmlConstants.java

```diff
@@ -141,13 +161,25 @@ public class HtmlConstants {
     "title",
     "tr",
     "track",
+    "tt", // deprecated
     "u",
     "ul",
     "var",
     "video",
-    "wbr"
+    "wbr",
+    "xmp" // deprecated
   );
 
+  // computed from https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/src/util/isNonInteractiveElement.js
+  public static final Set<String> NON_INTERACTIVE_ELEMENTS = Set.of("abbr", "address", "article", "aside", "blockquote", "br", "caption", "code", "dd", "del", "details", "dfn",
```

Comment:
Could you add another set for interactive elements?

Reply:
Isn't it simply?
```java
! NON_INTERACTIVE_ELEMENTS.contains(myTag)
```

Reply:
Indeed, never mind my comment.